src/locales/fr/dict.json

```diff
@@ -19,6 +19,15 @@
       "title": "Désolé, votre navigateur a généré une erreur"
     },
     "contactSupport": "Contacter le service clientèle",
+    "EMAIL_CHANGE_VERIFICATION_ERROR": {
+      "content": {
+        "email": "<0>Email :</0> helpdesk@autoscout24.ch",
+        "phone": "<0>Téléphone :</0> +41 31 744 17 37",
+        "workingHours": "<0>Horaires d'ouverture :</0> Lundi - Vendredi 08h00-12h00 / 13h00-17h00"
```

Comment:
I noticed we have different working hours format for different languages. 
Is this on purpose?